example.nxc

```diff
@@ -6,18 +6,28 @@
 task main()
 {
     SetSensorNXTViSy(PORT);
-    NXTViSyInit(SIGNATURE, 10, 24 * 100);
+    long err = NXTViSyInit(SIGNATURE, 10, 24 * 100);
+    if (err != NO_ERR) {
+        NumOut(0, LCD_LINE4, err);
+        while(1);
+    }
 
     int distance;
     byte action;
     string writable_action = "x";
 
     while (true) {
-        NXTViSyGetData(distance, action);
+        ClearScreen();
+        err = NXTViSyGetData(distance, action);
+        if (err != NO_ERR) {
+            NumOut(0, LCD_LINE5, err);
```

Comment:
@vargac same sa above, though I understand your intention with different lines for different errors.